samples/CallWithChat/src/app/views/HomeScreen.tsx

```diff
@@ -69,7 +69,7 @@ export const HomeScreen = (props: HomeScreenProps): JSX.Element => {
   const defaultDisplayName = localStorageAvailable ? getDisplayNameFromLocalStorage() : null;
   const [displayName, setDisplayName] = useState<string | undefined>(defaultDisplayName ?? undefined);
 
-  const [chosenCallOption, setChosenCallOption] = useState<IChoiceGroupOption>(callOptions[0]);
+  const [chosenCallOption, setChosenCallOption] = useState(callOptions[0] as IChoiceGroupOption);
```

Comment:
Is it possible this value could be still undefined? Maybe still check this value just in case we have a different type here and runtime (only when someone write a regression code, but hard to debug)

Could be something like this as well:
useState<IChoiceGroupOption|undefined>(callOptions[0]);

Reply:
Good point!

Reply:
Thanks that's much nicer, updated